server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider adding comments to explain the purpose of the variables PORT and HOSTNAME.